<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGprsPara</class>
 <widget class="QWidget" name="CGprsPara">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGprsPara</string>
  </property>
  <widget class="QPushButton" name="m_BtnCancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnOk">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnAdd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnDel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_pGprsTab">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>771</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>模式</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>地址1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>地址2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>APN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>用户</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>密码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>规约号</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
